### PR TITLE
LWDEV-8998 Use Spring mvc in httpOrderBook endpoint

### PR DESCRIPTION
**Pull request by rocketmannot**

### CODE REVIEW

File: src/main/kotlin/com/lykke/matching/engine/outgoing/rabbit/impl/RabbitMqPublisher.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        // MINIMAL_PERSISTENT_BASIC + headers
+        return AMQP.BasicProperties.Builder()
+                .deliveryMode(2)
+                .headers(headers)
+                .build()
+    }
+
+    private fun getLogMessage(item: Event<*>): String {
+        return gson.toJson(item)
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/matching/engine/outgoing/rabbit/impl/RabbitMqPublisher.kt]: Serialize to json only if it is needed to log (messageDatabaseLogger is not null)

File: src/main/kotlin/com/lykke/matching/engine/web/dto/StopOrderBook.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.util.*
+
+data class StopOrderBook(val assetPair: String,
+                         @get:JsonProperty("isBuy")
+                         @JsonProperty("isBuy")
+                         val isBuy: Boolean,
+
+                         @get:JsonProperty("isLower")
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/matching/engine/web/dto/StopOrderBook.kt]: Is it possible to configure boolean fields names policy during gson initialization ?

Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/web/dto/StopOrderBook.kt]: this is greate hack - to display isBuy in swagger examples - as springfox harcoded jackson  json serializer in their code - so this is annotation for jackson

File: src/main/kotlin/com/lykke/matching/engine/outgoing/rabbit/impl/RabbitMqPublisher.kt (hunk 1 above)
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/outgoing/rabbit/impl/RabbitMqPublisher.kt]: fixed

**Pull request merged**